src/Aspire.Hosting.Azure.KeyVault/api/Aspire.Hosting.Azure.KeyVault.cs

```diff
@@ -12,6 +12,20 @@ public static partial class AzureKeyVaultResourceExtensions
     {
         public static ApplicationModel.IResourceBuilder<Azure.AzureKeyVaultResource> AddAzureKeyVault(this IDistributedApplicationBuilder builder, string name) { throw null; }
 
+        public static ApplicationModel.IResourceBuilder<Azure.AzureKeyVaultSecretResource> AddSecret(this ApplicationModel.IResourceBuilder<Azure.AzureKeyVaultResource> builder, string secretName, ApplicationModel.IResourceBuilder<ApplicationModel.ParameterResource> parameterResource) { throw null; }
```

Comment:
In terms of consistency we should probably have:

AddSecret(builder, name, parameter)
AddSecret(builder, name, secretName, parameter)

... instead of:

AddSecret(builder, secretName, parameter)
AddSecret(builder, name, secretname, parameter)

???

Reply:
[PR #10521 - Standardize AddSecret extension method parameter naming](https://github.com/dotnet/aspire/pull/10521)